build.gradle.kts

```diff
@@ -22,7 +22,7 @@ plugins {
     id("java")
     id("org.springframework.boot") version "3.0.0"
     id("io.spring.dependency-management") version "1.1.0"
-    id("com.netflix.dgs.codegen") version "5.6.5"
```

Comment:
Please use the latestexplicit version instead of `latest.release` since this is not an internal app.